openstack/cinder/templates/backup_configmap.yaml

```diff
@@ -14,6 +14,8 @@ template: |{{`
       system: openstack
       type: configuration
       component: cinder
+      vcenter: {{ host }}
+      datacenter: {= availability_zone =}
```

Comment:
This is mixing two different ways of templating. I think it has to be `{{ availability_zone }}`

Reply:
Indeed,  missed that one.

Reply:
ah, this was confusing, the line above changed.